warehouse/accounts/forms.py

```diff
@@ -115,8 +117,15 @@ def validate_new_password(self, field):
             field.data, tags=["method:new_password"]
         ):
             raise wtforms.validators.ValidationError(
-                "This password has appeared in a breach or has otherwise been "
-                "compromised."
+                (
+                    "This password has appeared in a breach or has otherwise been "
+                    "compromised and cannot be used. See {project_help} for more "
```

Comment:
tried wrapping in an `a` tag, but we don't render html in our validation errors... which is pretty wise.

Reply:
Hrm. Maybe It would be better to raise a sigil here, like "<REPLACE WITH BREACHED PASSWORD HELP>", and add a conditional to the template looking for that string in specific? It's kind of gross but it lets us put real HTML in the error message?

Actually, what happens if you try to put html in this? I wonder if ``raise ValidationError(jinja2.Markup("..."))`` works?

Reply:
neat, your suggestion works! updating.